library/options.js.php

```diff
@@ -76,15 +76,26 @@ function checkSkipConditions() {
     var srcelem = document.getElementById('check_' + tofind);
     if (srcelem == null) srcelem = document.getElementById('radio_' + tofind);
     if (srcelem == null) srcelem = document.getElementById('form_' + tofind);
+    if (srcelem == null) srcelem = document.getElementById('text_' + tofind);
     if (srcelem == null) {
-      if (!cskerror) alert('<?php echo xls('Cannot find a skip source field for'); ?> "' + tofind + '"');
-      myerror = true;
-      continue;
+    if (!cskerror) alert('<?php echo xls('Cannot find a skip source field for'); ?> "' + tofind + '"');
+    myerror = true;
+    continue;
```

Comment:
Should maintain the spacing here of the lines within this conditional to maintain readabilty.